File: Ruby/lib/Cultist.rb
#encoding: utf-8

# Autores:
# 
#     Guillermo Montes Martos
#     Manuel Ruiz López
#

module NapakalakiGame
  class Cultist
    
    attr_reader :gainedLevels
    
    def initialize (name, gainedLevels)
      @name = name
      @gainedLevels = gainedLevels      
    end
    
  end
end

File: Java/src/NapakalakiGame/Cultist.java
/*
 * Autores:
 * Guillermo Montes Martos
 * Manuel Ruiz López
 *
 */
package NapakalakiGame;

public class Cultist {
    
    private String name;
    private int gainedLevels;
    
    
    // Constructor
    
    public Cultist(String name, int gainedLevels){
        this.name = name;
        this.gainedLevels = gainedLevels;
    }
    
    public int getGainedLevels(){
        return gainedLevels;
    }
}
